controllers/admin/AdminBlockListingController.php

```diff
@@ -168,9 +168,17 @@ public function displayAjaxSaveBlockContent()
         }
 
         if (strpos($picto, $this->module->img_path_perso) !== false) {
+            if ($picto) {
```

Comment:
I think it's better to check the value or if the variable is defined, instead of just put `if ($picto)`

Reply:
You're right, I replace with:

`if ($picto != '') {`